user/server-grpc-example.md

```diff
@@ -0,0 +1,156 @@
+# Example usage of the gRPC interface
+
+On the [previous section](server-protocol.md) we checked
+the format of the messages exchanged using the gRPC protocol. Here, we're
+going to put that knowledge into practice with a simple example usage of the Babelfish server using that protocol.
+
+This example is done in Go but you could use the gRPC interface using [any
+programming language that supports this protocol](http://www.grpc.io/about#osp).
+
+## Getting and compiling the `.proto` file
+
+The first step involving gRPC communication is getting the `.proto` file that
+defines the types involved in the communication. The [`.proto`
+format](https://developers.google.com/protocol-buffers/docs/proto) defines the
+structures and methods used for protocol buffer data. In the case of the Babelfish
+server protocol, you need to get [this `.proto`
+file](https://github.com/bblfsh/sdk/blob/master/protocol/generated.proto) from the
+SDK. Then you have to compile it to a source file with the required structures
+and methods. How you generate this module is language-dependent but usually it
+involves installing gRPC using your language package management tool (pip, cargo,
```

Comment:
True, I'll just remove the package managers and add "if it has a package for it".